src/main/kotlin/org/koitharu/kotatsu/parsers/site/madara/pt/DemonSectParser.kt

```diff
@@ -5,8 +5,8 @@ import org.koitharu.kotatsu.parsers.MangaSourceParser
 import org.koitharu.kotatsu.parsers.model.MangaSource
 import org.koitharu.kotatsu.parsers.site.madara.MadaraParser
 
-@MangaSourceParser("PRISMA_SCANS", "PrismaScans", "pt")
-internal class PrismaScansParser(context: MangaLoaderContext) :
-	MadaraParser(context, MangaSource.PRISMA_SCANS, "prismacomics.com", 10) {
+@MangaSourceParser("DemonSect", "DemonSect", "pt")
```

Comment:
`@MangaSourceParser("DEMONSECT", "DemonSect", "pt")`

Name of source (not a title) should be capitalized